src/app/app.module.ts

```diff
@@ -91,9 +94,11 @@ import { ExitSystemDialogComponent } from './components/navbar/dialogs/exit/exit
     MatDividerModule,
     MSWApolloModule,
     PagesModule,
+    PapaParseModule,
   ],
   declarations: [
     AppComponent,
+    ErrorDialogComponent,
```

Comment:
hey @ulmanA ,
it this a generic error dialog that it's being provided in app module?
And even if so, can you please move it to common or shared module
thanks